rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -494,6 +493,18 @@ public boolean equals(Object o) {
 		return equals;
 	}
 
+	static public class LocalDateTimeConverter implements DynamoDBTypeConverter<String, LocalDateTime> {
```

Comment:
Has this been tested with DynamoDB to see that it converts correctly?

Reply:
Yes I manually tested this.

Reply:
All I did was add a way for DynamoDb to know how to convert LocalDateTime to a string. But the attribute in DynamoDb itself is still a string so it still converts. I tested this with all 3 endpoints I have so far.